Frontend/ui-common/src/Engineering/Helpers/JBridge.js

```diff
@@ -1417,3 +1417,43 @@ export const goBackPrevWebPage = function (id) {
     console.log("goBackPrevWebPage error " + err);
   }
 }
+
+export const setCleverTapUserData = function (key) {
+  return function (value) {
+        console.log("setCleverTapUserData called in js 1");
+        if(window.JBridge.setCleverTapUserData){
+            console.log("setCleverTapUserData called in js 2");
+            window.JBridge.setCleverTapUserData(key, value);
+        }
+  };
+ };
+ 
+ export const setCleverTapUserProp = function (key) {
+   return function (value) {
+         console.log("setCleverTapUserProp called in js 1");
+         if(window.JBridge.setCleverTapUserProp){
+             console.log("setCleverTapUserData called in js 2");
+             window.JBridge.setCleverTapUserProp(key, value);
```

Comment:
if its always string define type for `value` as `String` else if its object use `JSON.stringify`